front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/_shared/BaseCarrousel/ScrollableCarrousel/index.tsx

```diff
@@ -92,7 +90,7 @@ const ScrollableCarrousel = ({
   }, [scrollContainerRef, hasMore, handleButtonVisiblity]);
 
   return (
-    <>
+    <Box position="relative">
```

Comment:
I'd actually prefer to move the SkipButton's absolute positioning styles in a Box wrapping the button here, so the relative/absolute relationship is clearer.

Similarly, I think the ScrollButtons could go inside HorizontalScroll so the positioning relationship could be made clearer.